addons/website/static/src/systray_items/edit_in_backend.js

```diff
@@ -23,7 +23,7 @@ export class EditInBackendSystray extends Component {
 
     editInBackend() {
         const { metadata: { mainObject } } = this.websiteService.currentWebsite;
-        this.websiteService.editedObjectPath = this.websiteService.contentWindow.location.pathname;
+        this.websiteService.editedObjectPath = this.websiteService.contentWindow.location.pathname + this.websiteService.contentWindow.location.search;
```

Comment:
We could also add the hash while at it :) 

```suggestion
        const location = this.websiteService.contentWindow.location;
        this.websiteService.editedObjectPath = location.pathname + location.search + location.hash;
```